doc/cabal-commands.rst

```diff
@@ -51,6 +51,143 @@ is in place, which moves the old configuration to a ``cabal.project.local~``
 file, this feature can also be disabled by using the ``--disable-backup``
 flag.
 
+.. option:: --default-user-config=file
```

Comment:
Here, we are in a subsection about `v2-configure`. Is there a section/subsection somewhere about commands common to all v2- commands?

Reply:
i can put them at the top section (https://cabal.readthedocs.io/en/3.6/cabal-commands.html)?

Reply:
Sure. Or create a distinct subsection --- whichever you choose. Now that I think about it, I'm not sure these are common to all v2-commands. There may be exceptions. Are there? If so, a couple of sentences before the command listing starts should say that they are common to many v2-commands, but to determine which belongs to which, one should use `cabal help cmd`.

Reply:
BTW, this links is better: https://cabal.readthedocs.io/en/latest

Reply:
I mean, the reader of the docs needs to experiment. Yes, moving is enough, with a sentence that says something, but not too much --- this is what I'm agonizing about.

Reply:
Perhaps just tweak "describing the arguments and flags they accept"? E.g., "first describing the arguments and flags that are common to some or all of them"? That implies that args and flags in subsections are not common.